modules/prelude-scala.el

```diff
@@ -33,10 +33,10 @@
 ;;; Code:
 
 (require 'prelude-programming)
-(prelude-require-packages '(scala-mode2))
+(prelude-require-packages '(ensime))
```

Comment:
The basic functions like syntax highlighting are automatically active when you install the package. And more complex functions do need manual init regarding which manage tool you are using.